packages/core-typings/src/ILivechatVisitor.ts

```diff
@@ -46,6 +62,7 @@ export interface ILivechatVisitor extends IRocketChatRecord {
 	contactManager?: {
 		username: string;
 	};
+	wrongMessageCount?: number;
```

Comment:
Same comment applies here as well -> https://github.com/RocketChat/Rocket.Chat/pull/29590#discussion_r1251566661 
Let's move this to room level as well